docs/markdoc-component-documentation.md

```diff
@@ -23,11 +23,11 @@ To see the rendered version of these tags, move this file into the `next/pages/g
 
 When the wire was despatched he had a cup of tea; over it he told me of a diary kept by Jonathan Harker when abroad, and gave me a typewritten copy of it, as also of Mrs. Harker's diary at Whitby. "Take these," he said, "and study them well. When I have returned you will be master of all the facts, and we can then better enter on our inquisition. Keep them safe, for there is in them much of treasure. You will need all your faith, even you who have had such an experience as that of to-day. What is here told," he laid his hand heavily and gravely on the packet of papers as he spoke, "may be the beginning of the end to you and me and many another; or it may sound the knell of the Un-Dead who walk the earth. Read all, I pray you, with the open mind; and if you can add in any way to the story here told do so, for it is all-important. You have kept diary of all these so strange things; is it not so? Yes! Then we shall go through all these together when we meet." He then made ready for his departure, and shortly after drove off to Liverpool Street. I took my way to Paddington, where I arrived about fifteen minutes before the train came in.
 
-## Complex Header Where You Definitely Want to Use an Anchor Link {% #complex-header %} 
+## Complex Header Where You Definitely Want to Use an Anchor Link {% #complex-header %}
```

Comment:
sorry for the random trailing whitespace removal in this file, I think it was automatically done by my editor (Zed)